xknx/devices/action.py

```diff
@@ -24,6 +24,7 @@ def test_counter(self, counter):
 
     def test_if_applicable(self, state, counter=None):
         """Test if should be executed for this state and this counter number."""
+        # pylint: disable=import-outside-toplevel
```

Comment:
Hm, i don't remember the reason why i made this a local import ...

Reply:
It's because of a circular dependency conflict. Im not really sure how to prevent this. New Stateupdater eliminates some of these.